.travis.yml

```diff
@@ -35,11 +35,9 @@ install:
  - true
 
 script:
-  - ./gradlew build idea -s -PcheckJava6Compatibility
+  - ./gradlew build idea -s -PcheckJava6Compatibility && ./gradlew ciPerformRelease
```

Comment:
Cant this just be `./gradlew build idea -s -PcheckJava6Compatibility ciPerformRelease` ?